cli.mjs

```diff
@@ -47,42 +47,57 @@ commander
     STUDY_ID = studyID;
     PARTICIPANT_ID = participantID;
   });
-commander.parse();
 
-// print message if download or delete provided, along with optional args provided
-if (ACTION != undefined) {
-  console.log(
-    `${ACTION} data from Firebase given ${STUDY_ID === undefined ? "" : `study ID: ${STUDY_ID}`} ${PARTICIPANT_ID === undefined ? "" : `and participant ID: ${PARTICIPANT_ID}`}`
-  );
-}
+// register: optional argument studyID and participantID skips relative prompts
+commander
+  .command(`register`)
+  .argument(`[studyID]`)
+  .argument(`[participantID]`)
+  .description(
+    `Register new partipant under study provided a partipantID and studyID; new study will be created if not found`
+  )
+  .action((studyID, participantID) => {
+    ACTION = "register";
+    STUDY_ID = studyID;
+    PARTICIPANT_ID = participantID;
+  });
+
+commander.parse();
 
 /** -------------------- MAIN -------------------- */
 
 // TODO @brown-ccv #289: Pass CLI arguments with commander (especially for action)
 async function main() {
```

Comment:
If you do something like this it should still parse the commander arguments correctly! It's just a little more explicit that that's the first_ thing we do when the CLI runs 
```suggestion

/** -------------------- MAIN -------------------- */

async function main() {
commander.parse();
```